style.css

```diff
@@ -671,3 +723,985 @@
+
+    /* identical to box height */
+
+    color: #fff;
+  }
+
+  .blue_box ul {
+    width: 178px;
+    height: 135px;
+    left: 78px;
+    top: 110px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 27px;
+    letter-spacing: -0.0525em;
+    color: #fff;
+  }
+
+  .blue_box img {
+    width: 102px;
+    height: 102px;
+    left: 186px;
+    top: 187px;
+  }
+
+  .brown_box {
+    width: 289px;
+    height: 289px;
+    left: 449px;
+    top: 3046px;
+    background: #e2505c;
+  }
+
+  .brown_box h3 {
+    width: 132px;
+    height: 30px;
+    left: 91px;
+    top: 52px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 20px;
+    line-height: 30px;
+
+    /* identical to box height */
+
+    color: #fff;
+  }
+
+  .brown_box ul {
+    width: 118px;
+    height: 135px;
+    left: 110px;
+    top: 110px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 27px;
+    letter-spacing: -0.0525em;
+    color: #fff;
+  }
+
+  .brown_box img {
+    width: 102.22px;
+    height: 102.22px;
+    left: 188px;
+    top: 187px;
+    opacity: 0.5;
+    transform: rotate(0.53deg);
+  }
+
+  .yellow_square {
+    width: 289px;
+    height: 289px;
+    left: 160px;
+    top: 3335px;
+    background: #febf52;
+  }
+
+  .yellow_square h3 {
+    width: 55px;
+    height: 30px;
+    left: 60px;
+    top: 52px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 20px;
+    line-height: 30px;
+
+    /* identical to box height */
+
+    color: #fff;
+  }
+
+  .yellow_square ul {
+    width: 88px;
+    height: 135px;
+    left: 78px;
+    top: 110px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 27px;
+    letter-spacing: -0.0525em;
+    color: #fff;
+  }
+
+  .yellow_square img {
+    width: 102px;
+    height: 102px;
+    left: 188px;
+    top: 187px;
+    opacity: 0.5;
+  }
+
+  .about h1 {
+    position: relative;
+    width: 295px;
+    height: 110px;
+    left: 834px;
+    top: 74px;
+    font-family: "Playfair Display", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 110px;
+    line-height: 100%;
+
+    /* identical to box height, or 110px */
+
+    color: #1a2236;
+  }
+
+  .about p {
+    position: relative;
+    width: 34vw;
+    height: 84px;
+    left: 834px;
+    top: 174px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    color: #1a2236;
+  }
+
+  .about button {
+    position: relative;
+    box-sizing: border-box;
+    width: 254px;
+    height: 70px;
+    left: 834px;
+    top: 240px;
+    color: #fff;
+    text-align: left;
+    padding: 14px 60px;
+    background: #1a2236;
+  }
```

Comment:
- [ ] Kindly make sure your design matches the Figms by making sur your get cv button is similar to the Figma's
Your design:
![Screenshot from 2023-03-04 19-18-47](https://user-images.githubusercontent.com/97834160/222919824-35841251-dc2d-49da-ba97-f400545f0d08.png)

Figma:
![Screenshot from 2023-03-04 19-06-03](https://user-images.githubusercontent.com/97834160/222919775-87d6a4aa-efeb-4cd0-b818-b542d7860ffe.png)